### DISCUSSION TITLE
我在微调代码的时候报错，不论是千问还是glm都是这个错误

### DISCUSSION THREAD

**Discussion by cca12138** [zai-org/ChatGLM3#1108] [Q&A] [answered]
![1712650345391(1)](https://github.com/THUDM/ChatGLM3/assets/150320711/db56da0e-e274-4e03-b044-160298067fd0)
Detected kernel version 5.4.0, which is below the recommended minimum of 5.5.0; this can cause the process to hang. It is recommended to upgrade the kernel to the minimum version or higher.

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
你内核太老了，要升级了
